content/en/blog/_posts/2022-12-06-fsgroup-on-mount.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: blog
+title: "Kubernetes 1.26: Providing pod's FSGroup to CSI drivers during mount"
+date: 2022-12-06
+slug: kubernetes-12-06-fsgroup-on-mount
+---
+
+**Author:** Fabio Bertinatto (Red Hat), Hemant Kumar (Red Hat)
+
+The delegation of fsGroup to CSI Drivers feature was first introduced as alpha in Kubernetes 1.22, reaching the beta mark in Kubernetes 1.25. Today we are happy to announce that this feature has graduated to General Availability (GA) in Kubernetes 1.26.
+
+Currently, if a fsGroup is specified in the [pod's security context](https://kubernetes.io/docs/tasks/configure-pod-container/security-context/#set-the-security-context-for-a-pod), all processes in the pod's containers will be part of the additional group specified in that field.
+
+In previous Kubernetes releases, the Kubelet would *always* apply the fsGroup ownership and permission changes to files in the volume according to the policy specified in the `Pod.spec.securityContext.fsGroupChangePolicy` field.
+
+Starting with Kubernetes 1.26, CSI drivers have the option to apply the fsGroup settings during attach or mount time of the volumes, freeing the Kubelet from changing the permissions of volumes.
```

Comment:
nit: we should drop "during attach" from the sentence. I don't see how this can be applied during attach.

Reply:
dropped